src/harness/reference_models/geo/utils.py

```diff
@@ -262,3 +262,22 @@ def PolygonsAlmostEqual(poly_ref, poly, tol_perc=10):
   intersection_polys = poly_ref.intersection(poly)
   return ((GeometryArea(union_polys) - GeometryArea(intersection_polys))
           < tol_perc/100. * GeometryArea(poly_ref))
+
+
+def hasPolygonCorrectGeoJsonWinding(polygon):
+  """
+  Args:
+    polygon: the reference GeoJSON geometry of  polygon object.
+
+   Returns: boolean set to true if the polygon has a correct GeoJson Winding and false otherwise.
+  """
+  shapely_polygon = utils.GeoJsonToShapelyGeometry(polygon);
```

Comment:
Note that we will have to make sure this function does not change the winding, like you implied in one of your example.
Because shapely may itself change the winding to another convention, and possibly depends on the implementation (ie the version of shapely used) as it is not clearly defined in the shapely spec.
Let me think and make some test about this early next week to be sure we have something robust on this matter: possibly we need to test every untouched LinearRing.

Reply:
ok thank you.
after finishing this, the test in test_polygon_correct_geoJson_winding should work fine.